lms/static/sass/course/layout/_banner_cta.scss

```diff
@@ -61,6 +61,12 @@
     .banner-cta-button {
       align-self: start;
       flex: none;
+      margin-top: $baseline;
+
+      @media only screen and (min-width: 630px) {
```

Comment:
We don't have a variable for that somewhere? I totally believe we don't, just a bummer.

Reply:
Yeah when I did a quick search for `@media only screen`, all the values were hardcoded :/